Sources/ChatUI/Essentials/GiphyConfiguration.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import GiphyUISDK
+import SwiftUI
+
+public class GiphyConfiguration {
+
+    let dimBackground: Bool
+    let showConfirmationScreen: Bool
+    let shouldLocalizeSearch: Bool
+    let mediaTypeConfig: [GiphyUISDK.GPHContentType]
+    let presentationDetents: CGFloat
```

Comment:
```suggestion
    public let dimBackground: Bool
    public let showConfirmationScreen: Bool
    public let shouldLocalizeSearch: Bool
    public let mediaTypeConfig: [GiphyUISDK.GPHContentType]
    public let presentationDetents: CGFloat
```
It would be better to access the values to read as public. Why don't we modify the access levels as a public🙂